contracts/MechMarketplace.sol

```diff
@@ -204,14 +204,12 @@ contract MechMarketplace is IErrorsMarketplace {
     /// @notice The request is going to be registered for a specified priority mech.
     /// @param requestDatas Set of self-descriptive opaque request data-blobs.
     /// @param priorityMechServiceId Priority mech service Id.
-    /// @param requesterServiceId Requester service Id, or zero if EOA.
```

Comment:
This came out as a parameter that can always be equal to zero, and it passes all the checks. So checks on requesters become obsolete.